Util/FormatNegotiator.php

```diff
@@ -11,134 +11,11 @@
 
 namespace FOS\RestBundle\Util;
 
-use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpFoundation\RequestMatcherInterface;
-use Negotiation\FormatNegotiator as BaseFormatNegotiator;
-use Negotiation\AcceptHeader;
+use FOS\RestBundle\Negotiation\FormatNegotiator as BaseFormatNegotiator;
 
-class FormatNegotiator implements MediaTypeNegotiatorInterface
+/**
+ * @deprecated since version 1.7, to be removed in 2.0. Use {@link BaseFormatNegotiator} instead.
```

Comment:
since this is going into master the deprecation notices should go into the 1.7 branch